packages/base/src/StaticArea.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @class
+ * @author SAP SE
+ * @private
+ * Static class for static area
+ */
+class StaticArea {
+	// Create a single instance of StaticArea
+	constructor() {
```

Comment:
I think the singleton pattern is more like:
```js
constructor() {
    if (!instance) { // if we created one instance, keep returning it all the time
      instance = this;
    }
    return instance;
  }
```
In this case, on first glance , the "if" statement would always return
and the code on line 14 can`t be reached, and finally multiple instances would be created.
Still, the code is safe as the `_getInstance `method ensures that one area in terms of DOM element will be created.

If nobody should do `new StaticArea` and everything you need is the `_getInstance()` to get the static area DOM, this might not be a class, but a function `getStaticArea`  and used instead of `StaticArea._getInstance()` in the files you need via `import { getStaticArea, destroyStaticArea } from StaticArea.js`.